pattern-1/bosh-release/jobs/wso2is/templates/ctl.erb

```diff
@@ -46,7 +46,11 @@ export WSO2_IS_SERVER_PACKAGE=/var/vcap/packages/wso2is/
 export MIN_HEAP=<%= properties.wso2is.memory.min_heap %>
 export MAX_HEAP=<%= properties.wso2is.memory.max_heap %>
 export MAX_PERMGEN=<%= properties.wso2is.memory.max_permgen %>
-export JAVA_OPTS=" -Xmx${MAX_HEAP}m -Xms${MIN_HEAP}m -XX:MaxPermSize=${MAX_PERMGEN}m"
+
+export JAVA_OPTS=" -Xmx${MAX_HEAP}m -Xms${MIN_HEAP}m \
+  -XX:MaxPermSize=${MAX_PERMGEN}m -Djdk.tls.ephemeralDHKeySize=2048 \
```

Comment:
As permanent generation has removed from java 8 onwards, do we still need `XX:MaxPermSize` OPT ?

Reply:
Fixed with https://github.com/wso2/pivotal-cf-is/pull/87/commits/fddbe789958f4a1ea6de0f9628746d54c777acdb